src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/ClientTradesRepository.cs

```diff
@@ -47,6 +48,22 @@ public TransactionStates State
         public double Volume { get; set; }
         public string ClientId { get; set; }
 
+        public double FeeSize { get; set; }
+        public FeeType FeeType { get; set; }
+        public string FeeTypeText
```

Comment:
Is this correct that both two properties are writable? I think one should be readonly as it derives from the second.